watertap/property_models/anaerobic_digestion/adm1_reactions.py

```diff
@@ -1325,7 +1325,7 @@ def Dissociation_rule(self, t):
         # Equation from [2]
         def CO2_acid_base_equilibrium_rule(self, t):
             return (
-                self.K_a_co2
+                self.K_a_co2 * 1e-2
```

Comment:
Any reason to manually scale the constraint here? We can always use the scaling constraint transform from IDAES

Reply:
No good reason. Was just checking values that worked and didn't clean up